### PR TITLE
Updated the README to include

### PR DESCRIPTION
**Pull request by cppwfs**
* contributor guidelines link
* issue tracking
* code of conduct
* license

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -28,19 +28,18 @@ yarn build          # Run dev
...
 
-### Algolia
+We welcome contributions! Follow this [link](https://github.com/spring-cloud/spring-cloud-dataflow/blob/master/spring-cloud-dataflow-docs/src/main/asciidoc/appendix-contributing.adoc) for more information on how to contribute.
 
-```bash
-export ALGOLIA_ADMIN_KEY=<KEY>
-```
+## Q&A and issue tracking
+
+If you have any feedback, or feature requests, don't hesitate to [add an issue](https://github.com/spring-io/dataflow.spring.io/issues).
```
Comment (sabbyanandan) [inline README.md]: Maybe, instead of "feature requests", we can rephrase it to include the guides or documentation as the context.

File: README.md [hunk 2]
```diff
@@ -28,19 +28,18 @@ yarn build          # Run dev
 yarn serve          # Serve the prod build
 ```
 
-## Configure
+## Contributing
 
-### Algolia
+We welcome contributions! Follow this [link](https://github.com/spring-cloud/spring-cloud-dataflow/blob/master/spring-cloud-dataflow-docs/src/main/asciidoc/appendix-contributing.adoc) for more information on how to contribute.
```
Comment (sabbyanandan) [inline README.md]: We could add a boilerplate to highlight that all the contributions are as simple as markdown files. Perhaps also point to the `./content/documentation` folder in a link, so folks can quickly get to it and relate to the website.

File: README.md [hunk 3]
```diff
@@ -6,7 +6,7 @@
 
 ## Introduction
 
-This is the **Spring Cloud Data Flow Website**, located at [https://dataflow.spring.io](https://dataflow.spring.io).
+This project contains the documentation and guides for the [dataflow.spring.io](https://dataflow.spring.io) site.
```
Comment (sabbyanandan) [inline README.md]: How about:

> This project contains the backing markdown files that get automatically generated as documentation and guides for the [Spring Cloud Data Flow Microsite](https://dataflow.spring.io) site accessible at  https://dataflow.spring.io.

File: README.md (hunk 2 above)
Comment (sabbyanandan) [reply README.md]: Also, I'm not sure pointing to SCDF's contribution guide would be useful for this repo, though. The content is kind of different in some ways.

**Comment by cppwfs** [comment]
Resolved issues, squashed, merged
